etc/testing/travis_build.sh

```diff
@@ -5,13 +5,16 @@ set -ex
 # We can't run the build step if there's no access to the secret env vars
 if [[ "$TRAVIS_SECURE_ENV_VARS" == "true" ]]; then
     docker login -u pachydermbuildbot -p "${DOCKER_PWD}"
-    make install docker-build
+    make install
     version=$(pachctl version --client-only)
+    git tag -f -am "Travis test v${version}" v"${version}"
```

Comment:
Just trying to understand this piece, what does `git tag` do here? Does `make docker-build` need it?